components/blocks/FilesToCards.vue

```diff
@@ -59,26 +61,229 @@
         </template>
       </DCard>
     </div>
+    <div v-else>
+      <div class="sort-bar">
+        <multiselect
+          class="item-a"
+          v-model="searchGroup"
+          :options="cardTags()"
```

Comment:
instead of using a grid here, maybe instead group each multiselect and it's button into a div, add a min-width on the multi-select, and make the outer div is-flex, is-flex-direction-row, flex-wrap-wrap.